### DISCUSSION TITLE
Empty

### DISCUSSION THREAD

**Discussion by Xu-Jayden** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1354] [General]
Empty

**Comment by MatthiasKerbl** [comment]
Hi @Xu-adam,

> The first one is that after I run calibration last week, my battery capacity drops. Is it common case?

Usually, it is more common to see an increase in capacity after running the Calibration Mode. However, it is also possible that the capacity decreases slightly.

> Another is that during the discharge stage of calibration, it really takes a very long time, very very long. Is it possible to add a feature that can speed up this process? I don't know whether it will do harm to batteries, I am just wondering.

We might do something about the long discharge period in the future. For now, a quick workaround would be to play a long youtube video for example.

Best,
Matthias
